java/ingest/ingest-core/src/main/java/sleeper/ingest/job/IngestJobSerDe.java

```diff
@@ -43,4 +59,13 @@ public String toJson(IngestJob ingestJob, boolean prettyPrint) {
     public IngestJob fromJson(String jsonIngestJob) {
         return gson.fromJson(jsonIngestJob, IngestJob.class);
     }
+
+    public Optional<IngestJob> fromJsonExpandingDirs(String jsonIngestJob) {
```

Comment:
If we move this method to the ingest-runner module, we can avoid using Hadoop and S3 in the core module.